src/App.tsx

```diff
@@ -15,36 +16,107 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+interface SortFieldAndIsReversed {
+  sortField: string;
+  isReversed: boolean;
+}
+
+const SORT_FIELD_ALPHABETICALLY = 'alphabetically';
+const SORT_FIELD_LENGHT = 'lenght';
+
+function getGoodsPrepared(
+  goods: string[],
+  { sortField, isReversed }: SortFieldAndIsReversed,
+) {
+  let preparedGoods = [...goods];
+
+  if (isReversed) {
+    preparedGoods = preparedGoods.reverse();
+  }
+
+  if (sortField) {
+    switch (sortField) {
+      case SORT_FIELD_ALPHABETICALLY:
+        return isReversed
+          ? preparedGoods.sort((good1, good2) => good2.localeCompare(good1))
+          : preparedGoods.sort((good1, good2) => good1.localeCompare(good2));
+
+      case SORT_FIELD_LENGHT:
+        return isReversed
+          ? preparedGoods.sort((good1, good2) => good2.length - good1.length)
+          : preparedGoods.sort((good1, good2) => good1.length - good2.length);
+
+      default:
+        return preparedGoods;
+    }
+  }
```

Comment:
```suggestion
    switch (sortField) {
      case SORT_FIELD_ALPHABETICALLY:
         preparedGoods.sort((good1, good2) => good1.localeCompare(good2));

      case SORT_FIELD_LENGHT:
         preparedGoods.sort((good1, good2) => good1.length - good2.length);

      default:
        
       }
  }
```
and simplify it here